.github/scripts/install-conda.sh

```diff
@@ -1,11 +1,12 @@
 # Actions invokes script with: bash -e <script>
 
```

Comment:
Just a bit more explicit: Use the conda install location specified in `common.sh`; do not rely on `cd`, but rather use explicit, absolute paths; and expand conda flags for better readability.